src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -163,11 +164,11 @@ const RegisterForm: React.FC<RegisterFormProps> = ({
 
       postRegisterCheck(auth, data);
 
-      const accountProfileUrl = `/account/profile${
-        spaceId ? `?spaceSlug=${spaceId}` : ""
```

Comment:
The original passes `spaceSlug` as a query parameter (after `?`).
The replacement `accountProfileUrl()` passes it as a path parameter (before `?`).

Has this been tested?

Reply:
fixed slug param + updated url util function
re-tested as well b50ddc5

Reply:
[NOTE]

Would have been better if this `/?spaceSlug=` was no longer the case, like, making it path than query param, and making it sure it works well as such.

Alas, due to time concerns, a `@debt` comment could be added about removing it or making a comprehensive redirect solution.